presentations/scopes/index.html

```diff
@@ -101,8 +101,7 @@
   <p>
   $$
     \begin{align}
-    X &\sqsubseteq Y \quad \text{// Read as } Y \text{ satisfies } X \\
-    &\phantom{\sqsubseteq \{} \text{// Derive tighter scope-sets} \\
```

Comment:
Why drop this line? it's a title for the next... Do make sure you view this locally, latex is easy to mess up :)